README.md

```diff
@@ -100,6 +100,47 @@ var client = ads.connect(options, function() {
 })
 ```
 
+
+### MultiRead something
+
+```javascript
+        this.client.multiRead(
+            [{
+                symname: '.TESTBOOL',
+                indexGroup: 16448,
+                indexOffset: 10438,
+                bytelength: ads.BOOL,
+            }, {
+                symname: '.TESTINT',
+                indexGroup: 16448,
+                indexOffset: 26938,
+                bytelength: ads.UINT,
+            }],
+            function (err, result) {
+                if (err) {
+                    console.error(err)
+                }
+                console.log(result)
```

Comment:
we should give a hint here on the err in the results.

Reply:
console.log(result)

result.forEach(function (handle) {
  if (handle.err) {
either    console.log(handle.err)
or         console.log(ads.errors[handle.err])
  }  else {
    console.log(handle)
  }
})